pkg/skaffold/watch/watch.go

```diff
@@ -73,16 +74,18 @@ func (w *watchList) Run(ctx context.Context, pollInterval time.Duration, onChang
 			return nil
 		case <-ticker.C:
 			changed := 0
-
+			var e Events
 			for i, component := range *w {
 				state, err := stat(component.deps)
 				if err != nil {
 					return errors.Wrap(err, "listing files")
 				}
+				e = events(component.state, state)
```

Comment:
Could we write `e := events(component.state, state)` here?

Reply:
Yes, leftover artifact of some refactoring, changed